python/Quickstart.MD

```diff
@@ -5,130 +5,107 @@ This quickstart covers:
+**Type or copy the following into your Python terminal**  
 ```
-{
-	"payload": {
-		"aptBusinessObjectSummary": "Incomplete shipment received.",
-		"aptBusinessObjectDescription": "Several packages missing or damaged.",
-		"deviationType": "UNPLANNED",
-		"materialType": "FINISHED_GOOD",
-		"materialSubtype": "SHIPPED",
-		"materialProblem": "SHORT_SHIPMENT_RECEIVED",
-		"isEscalated": false,
-		"createdByPartner": false,
-		"isSubmittedToPartner": false,
-		"directSupplierImpact": {
-			"businessImpact": "Business Impact",
-			"businessPriority": "HIGH"
-			}
-		}
+request_body = {
+  "header": {
+    "headerVersion": 1,
+    "eventName": "agile-process-teams:add-direct-supplier-incident:v3",
+    "ownerId": "YOUR_OWNER_ID",
+    "processNetworkId": "YOUR_PROCESS_NETWORK_ID",
+    "appName": "agile-process-teams",
+    "dataspace": "default"
+    },
+  "payload": {
+    "aptBusinessObjectSummary": "Incomplete shipment received.",
+    "aptBusinessObjectDescription": "Several packages missing or damaged.",
+    "deviationType": "UNPLANNED",
+    "materialType": "FINISHED_GOOD",
+    "materialSubtype": "SHIPPED",
+    "materialProblem": "SHORT_SHIPMENT_RECEIVED",
+    "isEscalated": False,
+    "createdByPartner": False,
+    "isSubmittedToPartner": False,
+    "directSupplierImpact": {
+      "businessImpact": "Business Impact",
+      "businessPriority": "HIGH"
+    }
+  }
 }
 ```
+This creates a dictionary object containing the `header` and `payload` elements we discussed earlier.  You can work with the items in this dictionary using standard Python methods.  This allows you to make any desired modifications prior to packaging and submitting your request.  
 
-The `utils.py` file has functions to help build the payload for your request body. The `read_payload_file(filename)` function reads the contents of the file, loads it into a JSON object, and does some minor processing for insertion into the request body.
+As an example, let's change the priority of this incident from `HIGH` to `MEDIUM`.  To change this value, we must reference the `businessPriority` item.  To reach this value, reference the `directSupplierImpact` element of the `payload`.  
 
-In your Python terminal, enter the following:  
+**Type or copy the following into your Python terminal**
 ```
->>> import utils
->>> my_file = 'addIncident.json'
->>> payload_data = utils.read_payload_file(my_file)
+request_body.get('payload').get('directSupplierImpact').update({'businessPriority': 'MEDIUM'})
 ```
-If you already use a JSON payload as part of your other processes, you can the `accept_json_payload(payload_in)` function instead. This function accepts a JSON object, checks for proper format, and does some minor processing for insertion into the request body.
+We use the `.get()` method for each element in our tree that we wish to access.  The `.update()` method allows us to change a value by referencing its key.
```

Comment:
eliminate we and us

"Use the `.get()` method for each element you wish to access. The `.update()` method allows for changing a value by referencing its key."